{"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2018-18748", "ASSIGNER": "cve@mitre.org"}, "affects": {"vendor": {"vendor_data": []}}, "problemtype": {"problemtype_data": [{"description": []}]}, "references": {"reference_data": [{"url": "https://github.com/sandboxescape/Sandboxie-5.26-Sandbox-Escape-Exploit/blob/2632a5f7409e52b2e020f5d4467fa019f9095e73/README.doc", "name": "https://github.com/sandboxescape/Sandboxie-5.26-Sandbox-Escape-Exploit/blob/2632a5f7409e52b2e020f5d4467fa019f9095e73/README.doc", "refsource": "MISC", "tags": []}]}, "description": {"description_data": [{"lang": "en", "value": "** DISPUTED ** Sandboxie 5.26 allows a Sandbox Escape via an 'import os' statement, followed by os.system('cmd') or os.system('powershell'), within a .py file. NOTE: the vendor disputes this issue because the observed behavior is consistent with the product\"s intended functionality."}]}}
